ros_buildfarm/templates/release/rpm/binarypkg_job.xml.em

```diff
@@ -123,6 +123,9 @@ but disabled since the package is blacklisted (or not whitelisted) in the config
         'echo "# BEGIN SECTION: Run Dockerfile - build binaryrpm"',
         'rm -fr $WORKSPACE/binarypkg',
         'mkdir -p $WORKSPACE/binarypkg/source',
+    ] + ([
+        'if [ ! -d "$HOME/.ccache" ]; then mkdir $HOME/.ccache; fi',
+    ] if shared_ccache else []) + [
```

Comment:
I noticed that this snip was missing from the RPM jobs. The snip prevents a root-owned `~/.ccache` directory from getting created.